Source/WebCore/page/scrolling/mac/ScrollingTreeScrollingNodeDelegateMac.mm

```diff
@@ -81,6 +81,10 @@
             m_scrollerPair->horizontalScroller().setHostLayer(nullptr);
         if (scrollingStateNode.scrollableAreaParameters().verticalNativeScrollbarVisibility != NativeScrollbarVisibility::Visible)
             m_scrollerPair->verticalScroller().setHostLayer(nullptr);
+
+        // TODO: Find a way to make this not run when UISideCompositing is off because it breaks initial scrollbar rendering
+        // TODO: Potentially should only run this when ScrollbarWidth updates specifically but I don't think we have an old copy of the scrollingStateNode to diff
+        m_scrollerPair->updateScrollbarWidth();
```

Comment:
When UISide Compositing is disabled this breaks the rendering of scrollbars on first load. The moment they're scrolled or you focus a different window they render correctly.

![image](https://github.com/WebKit/WebKit/assets/32498324/585b2716-dbd2-4657-abc7-c580c8484b58)